src/plugins/spdk/spdk_env/threads.c

```
@@ -0,0 +1,517 @@
+		if (strncmp(ent->d_name, "node", 4) == 0 &&
+		    ent->d_name[4] >= '0' && ent->d_name[4] <= '9') {
+			nid = (int32_t)strtol(ent->d_name + 4, NULL, 10);
+			break;
+		}
+	}
+
+	closedir(dir);
+	return nid;
+}
+
+int32_t spdk_env_get_first_numa_id(void)
```

Cppcheck (2.21.0)

[style] The function 'spdk_env_get_first_numa_id' is never used.

(unusedFunction)